File: app/src/main/java/org/ionc/wallet/callback/OnDeviceDetailCallback.java
package org.ionc.wallet.callback;


import org.ionc.wallet.bean.DeviceBean;

/**
 * USER: binny
 * DATE: 2018/9/15
 * 描述:
 */
public interface OnDeviceDetailCallback {
    /**
     * 请求设备详情成功
     *
     * @param deviceBean 设备信息
     */
    void onDeviceDetailSuccess(DeviceBean.DataBean deviceBean);

    /**
     * 请求设备详情失败
     */
    void onDeviceDetailFailure();
}

File: app/src/main/java/org/ionchain/wallet/mvp/callback/OnDeviceDetailCallback.kt
package org.ionchain.wallet.mvp.callback

import org.ionchain.wallet.bean.DeviceBean

/**
 * USER: binny
 * DATE: 2018/9/15
 * 描述:
 */
interface OnDeviceDetailCallback {
    /**
     * 请求设备详情成功
     * @param deviceBean 设备信息
     */
    fun onDeviceDetailSuccess(deviceBean: DeviceBean.DataBean)

    /**
     * 请求设备详情失败
     */
    fun onDeviceDetailFailure()
}
